### PR TITLE
Fixes #13399 : Add unit tests to the "TeamGroupStatusCmd" mmctl command

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
#### Summary
Added unit tests for checking team group status command

#### Ticket Link
Fixes : https://github.com/mattermost/mattermost-server/issues/13399
Jira : https://mattermost.atlassian.net/browse/MM-21240

### CODE REVIEW

File: commands/group_test.go [hunk 1]
```diff
@@ -166,3 +166,99 @@ func (s *MmctlUnitTestSuite) TestTeamGroupEnableCmd() {
...
+
+		err := teamGroupStatusCmdF(s.client, cmd, args)
+
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetErrorLines(), 0)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().Equal(printer.GetLines()[0], "Disabled")
+	})
+
+	s.Run("Should show valid response when status of team group status is enabled", func() {
```
Comment (streamer45) [inline commands/group_test.go]: Here I would use:  
`Should show valid response when group constraints status for a team is enabled`

File: commands/group_test.go [hunk 2]
```diff
@@ -166,3 +166,99 @@ func (s *MmctlUnitTestSuite) TestTeamGroupEnableCmd() {
...
+
+		err := teamGroupStatusCmdF(s.client, cmd, args)
+
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetErrorLines(), 0)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().Equal(printer.GetLines()[0], "Enabled")
+	})
+
+	s.Run("Should show valid response when status of team group status is disabled", func() {
```
Comment (streamer45) [inline commands/group_test.go]: And here:  
`Should show valid response when group constraints status for a team is disabled`

File: docs/mmctl_user_email.md [hunk 3]
```diff
@@ -13,8 +13,7 @@ mmctl user email [user] [new email] [flags]
 ### Examples
 
 ```
-  user email test user@example.com
-  user activate username
```
[the remark this answers is not in the document]
Comment (M-ZubairAhmed) [reply docs/mmctl_user_email.md]: this was auto generated when i ran `make docs`, do you want me to remove this. But i guess someone needs to run that command to keep docs updated sometime later too

[the remark this answers is not in the document]
Comment (streamer45) [reply docs/mmctl_user_email.md]: It's fine to leave it, just making sure it wasn't by mistake. Thanks for clarifying :+1:

**Comment by jespino** [comment]
Merged! thanks @M-ZubairAhmed! :tada:

**Pull request merged**